org.codeberg.dimkard.glossaico.json

```diff
@@ -0,0 +1,62 @@
+{
+    "id": "org.codeberg.dimkard.glossaico",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
```

Comment:
```suggestion
    "runtime-version": "5.15-21.08",
```

Reply:
I tried this first, but pyside2 did not build for me, IIRC due to something related to clang.